src/main/java/io/d11/aerospike/client/impl/AerospikeClientImpl.java

```diff
@@ -31,12 +31,15 @@
 import io.d11.aerospike.listeners.RecordListenerImpl;
 import io.d11.aerospike.listeners.WriteListenerImpl;
 import io.d11.aerospike.util.SharedDataUtils;
+import io.reactivex.rxjava3.core.Observable;
```

Comment:
This client should not depend on rxjava. This should use core vertx objects and rxfied code should be generated for both rxjava2 and rxjava3 using vertx gen